src/server/helpers/middleware.ts

```diff
@@ -32,25 +32,24 @@ interface $Request extends Request {
 	user: any
 }
 
-function makeLoader(modelName, propName, sortFunc?, relations = []) {
-	return function loaderFunc(req: $Request, res: $Response, next: NextFunction) {
-		const {orm}: any = req.app.locals;
-		const model = orm[modelName];
-		return model.fetchAll({withRelated: [...relations]})
-			.then((results) => {
-				const resultsSerial = results.toJSON();
-
-				res.locals[propName] =
-					sortFunc ? resultsSerial.sort(sortFunc) : resultsSerial;
-
-				next();
-
-				return null;
-			})
-			.catch(next);
-	};
+async function makeLoader(modelName, propName, sortFunc?, relations = []) {
+	try {
+		return async function loaderFunc(req: $Request, res: $Response, next: NextFunction) {
+			const {orm}: any = req.app.locals;
+			const model = orm[modelName];
+			const results = await model.fetchAll({withRelated: [...relations]});
+			const resultsSerial = results.toJSON();
+			res.locals[propName] =
+				sortFunc ? resultsSerial.sort(sortFunc) : resultsSerial;
+			next();
+			return null;
+		};
+	} catch (error) {
+		throw error;
```

Comment:
This doesn't make sense, why should you catch an error just to immediately throw it again?
As far as I can see this would still crash the server, the equivalent of the old code should be (untested):
```suggestion
		next(error);
```